src/app/setup-wizard/data-collection/on-site-assessment/neb-forms-accordion/neb-setup-form/performance-metrics-modal/metric-options-modal-list.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { KeyPerformanceMetric, KeyPerformanceMetricOption } from 'src/app/shared/constants/keyPerformanceMetrics';
+import * as _ from 'lodash';
+
+@Pipe({
+  name: 'metricOptionsModalList'
+})
+export class MetricOptionsModalListPipe implements PipeTransform {
+
+  transform(options: Array<KeyPerformanceMetricOption | KeyPerformanceMetric>, searchStr: string, orderByDir: 'asc' | 'desc'): Array<KeyPerformanceMetric> {
```

Comment:
should the return type be Array<KeyPerformanceMetricOption | KeyPerformanceMetric> for mixed types or Array<KeyPerformanceMetricOption> (since KeyPerformanceMetric extends KeyPerformanceMetricOption)?

Reply:
Yup. Thanks